### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Hedy/Adventures](https://hosted.weblate.org/projects/hedy/adventures/).

It also includes following components:

* [Hedy/Webpages](https://hosted.weblate.org/projects/hedy/webpages/)

* [Hedy/slides](https://hosted.weblate.org/projects/hedy/slides/)

* [Hedy/Parsons](https://hosted.weblate.org/projects/hedy/parsons/)

* [Hedy/Achievements](https://hosted.weblate.org/projects/hedy/achievements/)

* [Hedy/client-messages](https://hosted.weblate.org/projects/hedy/client-messages/)

* [Hedy/Keywords](https://hosted.weblate.org/projects/hedy/keywords/)

* [Hedy/Cheatsheets](https://hosted.weblate.org/projects/hedy/commands/)

* [Hedy/Quizzes](https://hosted.weblate.org/projects/hedy/quizzes/)

* [Hedy/Texts](https://hosted.weblate.org/projects/hedy/web-texts/)

* [Hedy/Tutorials](https://hosted.weblate.org/projects/hedy/tutorials/)

Current translation status:

Weblate translation status

### CODE REVIEW

**Comment by etienneCharignon** [comment]
fix #4715

**Comment by Felienne** [comment]
> fix #4715 (In French, echo and repeat are translated the same way)

Thank you!!

File: content/quizzes/id.yaml [hunk 1]
```diff
@@ -2013,26 +2013,26 @@ levels:
...
             -   option: '`{if}` `{else}` `{repeat}` `{print}`'
-                feedback: Not with print
-            hint: Indentation happens on the line below some commands
+                feedback: Tidak dengan {print}
+            hint: Indentasi terjadi pada baris di bawah beberapa perintah
             correct_answer: C
             question_score: '10'
         7:
-            question_text: "In this code from a pizza restaurant. \nYoull get a 5 dollar discount if you order a medium pizza with coke.\n What should you do to debug this code?"
+            question_text: "Anda akan mendapat diskon 5 dolar jika memesan pizza ukuran sedang dengan minuman bersoda.<br>Tetapi kodenya salah! Bagaimana cara men-debugnya?"
```
Comment (Felienne) [inline content/quizzes/id.yaml:2035]:
```suggestion
            question_text: "Anda akan mendapat diskon 5 dolar jika memesan pizza ukuran sedang dengan minuman bersoda. \nTetapi kodenya salah! Bagaimana cara men-debugnya?"
```

**Review by Felienne — APPROVED** [review]
Merging so we can move along Marleen's changes

**Comment by Felienne** [comment]
Oh no, Babel does not understand Pap (for Papiamento) 😭 

image

We will have to make another mapping I think, will dive in

**Pull request merged**